libgadget/density.c

```diff
@@ -534,10 +532,10 @@ void density_check_neighbours (int i, TreeWalk * tw) {
 
 #ifdef BLACK_HOLES
         if(P[i].Type == 5)
-            if(DENSITY_GET_PRIV(tw)->Left[i] > All.BlackHoleMaxAccretionRadius)
+            if(P[i].Left > All.BlackHoleMaxAccretionRadius)
```

Comment:
Left[i] and Right[i]. These are still the private members.

Reply:
Bah, that's what I get for ever removing BLACK_HOLES from my Options.mk